MicroServices/ShopFlowPro-service/src/main/java/com/wsu/shopflowproservice/utilities/Constants.java

```diff
@@ -0,0 +1,13 @@
+package com.wsu.shopflowproservice.utilities;
+
+import lombok.experimental.UtilityClass;
+
+@UtilityClass
+public class Constants {
+
+    public static final String MESSAGE = "message";
+    public static final String DESC = "desc";
+    public static final String PAGE_COUNT = "pageCount";
+    public static final String RESULT_COUNT = "resultCount";
+    public static final String ERROR_MESSAGE = "message";
```

Comment:
You have same value for this field and MESSAGE field which create duplicate key issue while building meta Map in ControllerExceptionHandler class. Change value for this field as errorMessage